package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "start": "node ./node_modules/react-native/local-cli/cli.js start",
     "test": "jest",
-    "postinstall": "ln -sf lib/{ios,android} ."
+    "postinstall": "ln -sf lib/ios . && ln -sf lib/android ."
```

Comment:
This seems like it should work. Or similar alternative using semi-colon instead of double-ampersand?